src/covid_model_seiir_pipeline/pipeline/forecasting/task/beta_forecast.py

```diff
@@ -24,6 +24,25 @@ def run_beta_forecast(forecast_version: str, scenario: str, draw_id: int):
     scenario_spec = forecast_spec.scenarios[scenario]
     data_interface = ForecastDataInterface.from_specification(forecast_spec)
 
+    #################
+    # Build indices #
+    #################
+    # The hardest thing to keep consistent is data alignment. We have about 100
+    # unique datasets in this model and they need to be aligned consistently
+    # to do computation.
+    logger.info('Loading index building data', context='read')
+    regression_start_dates = data_interface.load_regression_start_dates(draw_id)
+    forecast_start_dates = data_interface.load_forecast_start_dates(draw_id)
+    # Forecast is run to the end of the covariates
+    covariates = data_interface.load_covariates(scenario_spec)
+    forecast_end_dates = covariates.reset_index().groupby('location_id').date.max()
+    logger.info('Building indices', context='transform')
+    indices = model.Indices(
```

Comment:
so it's intentional that these `indices` are not yet used by the "old system".

Why not delay creating the instance until you're ready to use it? Just to see if creating it errors?